src/tests/run.cmd

```diff
@@ -200,7 +199,7 @@ echo.
 echo where:
 echo.
 echo./? -? /h -h /help -help   - View this message.
-echo ^<build_architecture^>      - Specifies build architecture: x64, x86, arm, or arm64 ^(default: x64^).
+echo ^<build_architecture^>      - Specifies build architecture: x64, x86, or arm64 ^(default: x64^).
```

Comment:
Hmm, I'm still seeing linux_arm / linux_musl_arm jobs in the above pipelines, are we removing arm32 support even on these?

Reply:
No, linux-arm is still a supported platform. (But run.cmd is only used on Windows)